OtripleS.Web.Api.Tests.Unit/Services/Foundations/GuardianContacts/GuardianContactServiceTests.Validations.RetrieveById.cs

```diff
@@ -54,7 +54,7 @@ public async Task ShouldThrowValidatonExceptionOnRetrieveWhenGuardianIdIsInvalid
         public async Task ShouldThrowValidatonExceptionOnRetrieveWhenContactIdIsInvalidAndLogItAsync()
         {
             // given
-            Guid randomContactId = default;
+            Guid invalidContactId = Guid.Empty;
```

Comment:
Make sure when change these variable names that you apply the same change wherever this variable is being used.

Reply:
@hassanhabib Changed all corresponding variables from "random" to "invalid".

Reply:
We should just need to use invalidGuardianId.